tests/upgrades/test_subscription.py

```diff
@@ -47,19 +42,19 @@ def test_pre_manifest_scenario_refresh(self, request):
 
         :expectedresults: Manifest should be uploaded and refreshed successfully.
         """
-        org = entities.Organization(name=f"{request.node.name}_org").create()
-        upload_manifest(settings.fake_manifest.url['default'], org.name)
-        history = entities.Subscription(organization=org).manifest_history(
-            data={'organization_id': org.id}
-        )
-        assert f"{org.name} file imported successfully." == history[0]['statusMessage']
-        sub = entities.Subscription(organization=org)
+        org = upgrade_entitlement_manifest_org
+        target_sat.cli.Subscription.list({'organization-id': org.id}, per_page=None)
+        history = target_sat.cli.Subscription.manifest_history({'organization-id': org.id})
+        assert f'{org.name} file imported successfully.' in ''.join(history)
```

Comment:
The missing space can also fail the string assertion here.
```suggestion
        assert f'{org.name} file imported successfully.' in ' '.join(history)
```